DayInMonth.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+
+if __name__ == '__main__':
+    print("Number of days, June, 1996: ")
+    print(np.datetime64("2016-07-01") - np.datetime64('2016-06-01'))
```

Comment:
What if we wanted to create a function that would allow counting the number of days in a given month (as an argument)?